packages/drivers/odsp-driver/src/odspFluidFileLink.ts

```diff
@@ -126,9 +128,13 @@ export function storeLocatorInOdspUrl(url: URL, locator: OdspFluidDataStoreLocat
  * Extract ODSP Fluid data store locator object from given ODSP url. This extracts things like
  * driveId, ItemId, siteUrl etc from a url where these are encoded in nav query param.
  * @param url - ODSP url representing Fluid file link
+ * @param requireFluidSignature - flag representing if the fluid signature is expected in the url, optional
```

Comment:
Nit: "optional" doesn't indicate whether the default behavior is true or false. I would note that the default here is "false".

Reply:
Hey, I updated the comment here and made the param default true.

Reply:
Thanks!